cpp/inc/bond/ext/grpc/unary_call.h

```diff
@@ -35,61 +28,36 @@ class shared_unary_call;
 /// @note This class can only be moved. If shared ownership semantics are
 /// needed, convert this to a \ref shared_unary_call with the \ref share
 /// function.
-///
-/// @warning A moved-from unary_call is only valid for destruction or
-/// assignment.
-template <typename TRequest, typename TResponse>
-class unary_call final : public detail::unary_call_base<TRequest, TResponse>
+template <typename Request, typename Response>
+class unary_call final : public detail::unary_call_base<Request, Response>
 {
-    using base_type = detail::unary_call_base<TRequest, TResponse>;
+    using base_type = typename unary_call::unary_call_base;
 
 public:
-    /// @brief The default constructor of unary_call is not callable: the
```

Comment:
Can we leave this documentation comment? It's become some what of a FAQ ("Why can't I default construct a `unary_call` instance?"), as move-only types aren't (yet) broadly familiar types. We can put it behind `BOND_DOXYGEN_ONLY`.